src/transforms/filter.js

```diff
@@ -15,7 +15,7 @@ var axisIds = require('../plots/cartesian/axis_ids');
 var autoType = require('../plots/cartesian/axis_autotype');
 var setConvert = require('../plots/cartesian/set_convert');
 
-var INEQUALITY_OPS = ['=', '<', '>=', '>', '<='];
+var INEQUALITY_OPS = ['=', '!=', '<', '>=', '>', '<='];
```

Comment:
hmm, nonblocking as this isn't new, but seems funny to categorize `'='` in `INEQUALITY_OPS`. How about `COMPARISON_OPS`?